<launch>
  <node name="nmpc" pkg="fw_ctrl" type="fw_nmpc"/>
  <param name="/nmpc/loop_rate" type="float" value="20.0" />
</launch>